src/runtime/audience-action-local-flow.ts

```diff
@@ -439,10 +449,24 @@ export class AudienceActionLocalFlow implements AudienceActionFlow {
       toArray: () => [],
       label: String,
     };
-    this.fetcher_.sendPost(url, emptyMessage);
-    // TODO: mhkawano - log error
-    // TODO: mhkawano - handle entitlement consumption logic on completion
-    // ex: this.entitlementsManager_.getEntitlements();
+    const response = (await this.fetcher_.sendPost(
+      url,
+      emptyMessage
+    )) as unknown as CompleteAudienceActionResponse;
+    if (response.updated) {
```

Comment:
Add test for the case that updated is false?

Reply:
Done.